README.md

```diff
@@ -49,6 +49,31 @@ Each benchmark will run until the target quality is reached and then stop, print
 
 Some these benchmarks are rather slow or take a long time to run on the reference hardware (i.e. 16 CPUs and one P100). We expect to see significant performance improvements with more hardware and optimized implementations. 
 
+
+# Running Instructions
+
+1. Clone the repository.
```

Comment:
Before listing the steps, it would be useful to have a small paragraph describing what Popper is and what it does.

Reply:
and also a quick description of what the `settings.py` do